waspc/cli/Command/Common.hs

```diff
@@ -0,0 +1,38 @@
+module Command.Common
+    ( findWaspProjectRootFromCwd
+    , findWaspProjectRoot
+    ) where
+
+import System.Directory (getCurrentDirectory, doesPathExist, doesFileExist)
+import qualified System.FilePath as FP
+import Data.Maybe (fromJust)
+import Control.Monad.Except (throwError)
+import Control.Monad.IO.Class (liftIO)
+import Control.Monad (when, unless)
+
+import StrongPath (Path, Abs, Dir)
+import qualified StrongPath as SP
+import Command (Command, CommandError(..))
+import Common (WaspProjectDir, dotWaspRootFileInWaspProjectDir)
+
+
+findWaspProjectRoot :: Path Abs (Dir ()) -> Command (Path Abs (Dir WaspProjectDir))
+findWaspProjectRoot currentDir = do
+    let absCurrentDirFp = SP.toFilePath currentDir
+    doesCurrentDirExist <- liftIO $ doesPathExist absCurrentDirFp
+    unless doesCurrentDirExist (throwError notFoundError)
+    let wasprootPath = absCurrentDirFp FP.</> SP.toFilePath dotWaspRootFileInWaspProjectDir
```

Comment:
```suggestion
    let waspRootPath = absCurrentDirFp FP.</> SP.toFilePath dotWaspRootFileInWaspProjectDir
```

Reply:
Good point, I actually renamed it to dotWaspRootFilePath!